x-pack/plugins/newsfeed/public/plugin.tsx

```diff
@@ -41,7 +41,10 @@ export class NewsfeedPublicPlugin implements Plugin<Setup, Start> {
     const { http } = core;
     const api$ = getApi(http).pipe(
       takeUntil(this.stop$), // stop the interval when stop method is called
-      catchError(() => {
+      tap(fetchResults => {
+        sessionStorage.setItem('newsfeed.debug', JSON.stringify(fetchResults));
```

Comment:
This is just for integration testing and debug - it will be removed before merge